code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -190,29 +190,44 @@
 	var/datum/language/species_language = GLOB.all_languages[language]
 	return species_language.get_random_name(gender)
 
-/datum/species/proc/create_organs(mob/living/carbon/human/H) //Handles creation of mob organs.
+
+/**
+ * Handles creation of mob organs.
+ *
+ * Arguments:
+ * * H: The human to create organs inside of
+ * * bodyparts_to_omit: Any bodyparts in this list (and organs within them) should not be added.
+ */
+/datum/species/proc/create_organs(mob/living/carbon/human/H, list/bodyparts_to_omit) //Handles creation of mob organs.
 	QDEL_LIST_CONTENTS(H.internal_organs)
 	QDEL_LIST_CONTENTS(H.bodyparts)
 
 	LAZYREINITLIST(H.bodyparts)
 	LAZYREINITLIST(H.bodyparts_by_name)
 	LAZYREINITLIST(H.internal_organs)
 
-	for(var/limb_type in has_limbs)
-		var/list/organ_data = has_limbs[limb_type]
+	for(var/limb_name in has_limbs)
+		if(bodyparts_to_omit && (limb_name in bodyparts_to_omit))
+			H.bodyparts_by_name[limb_name] = null  // Null it out, but leave the name here so it's still "there"
+			continue
+		var/list/organ_data = has_limbs[limb_name]
 		var/limb_path = organ_data["path"]
 		var/obj/item/organ/O = new limb_path(H)
 		organ_data["descriptor"] = O.name
 
 	for(var/index in has_organ)
-		var/organ = has_organ[index]
-		// organ new code calls `insert` on its own
-		new organ(H)
+		var/obj/item/organ/internal/organ_path = has_organ[index]
+		if(initial(organ_path.parent_organ) in bodyparts_to_omit)
+			continue
+
+		new organ_path(H)
```

Comment:
This right here is the key change between the first PR. It had previously been
```
var/obj/item/organ/internal/org = new organ_path()
org.insert(H)
```
which makes sense, but positronic brains qdel themselves on initialization without a brainmob. This means that every IPC's brain would become a ticking timebomb, waiting for the garbage subsystem to decide it's been around long enough to force it to harddel. When this would happen, it'd null out the brain in the body, which caused the key problems.

Reply:
could you add in a comment here, noting why you're specifically not using `insert()` (i.e. qdel issue)